src/style/material-icons.scss

```diff
@@ -0,0 +1,28 @@
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Material Icons'),
+    local('MaterialIcons-Regular'),
+    url(https://example.com/MaterialIcons-Regular.woff2) format('woff2'),
```

Comment:
Quotes like in `index.scss`?

Reply:
I don't think this PR should contain anything like icon fonts.